src/network/networkd-setlink.h

```diff
@@ -11,6 +11,7 @@ typedef enum SetLinkOperation {
         SET_LINK_BOND,                    /* Setting bond configs. */
         SET_LINK_BRIDGE,                  /* Setting bridge configs. */
         SET_LINK_BRIDGE_VLAN,             /* Setting bridge vlan configs. */
+        SET_LINK_CAN,                     /* Setting can interface configs. */
```

Comment:
given that CAN is an acronym, maybe uppercase it? (same for "vlan" in the line prior)